tdrs-backend/tdpservice/scheduling/sftp_task.py

```diff
@@ -78,8 +78,8 @@ def create_dir(directory_name, sftp_server):
                           pkey=pkey,
                           username=username,
                           port=port,
-                          look_for_keys=False,
-                          disabled_algorithms={'pubkeys': ['rsa-sha2-512', 'rsa-sha2-256']})
```

Comment:
This is only temporary for using AWS EC2 service. ACFTITAN server doesn't have RSA handshake and as such we should remove these settings.